content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -67,6 +66,13 @@ Endpoint Slices can act as the source of truth for kube-proxy when it comes to
 how to route internal traffic. When enabled, they should provide a performance
 improvement for services with large numbers of endpoints.
 
+## Address Types
+
+EndpointSlices support three address types:
+* IPv4
```

Comment:
nit: Insert a new line at line 72. This list does not render.

Reply:
Fixed, thanks for catching that!

Reply:
👍